power_measurement.adoc

```diff
@@ -111,93 +107,110 @@ If you are measuring the performance of an experimental framework or system, you
+https://github.com/mlcommons/inference_policies/blob/master/inference_rules.adoc[The MLPerf Inference rules]
+specify several phases of a benchmark: accuracy, performance, compliance.
+Power is evaluated only in the performance phase, and not in any other phases.
 
-The submission process has to use the software flow and scripts
-developed as part of the MLPerf benchmark Power measurement. The
-infrastructure has been developed by the MLPerf Power working group.
+=== LoadGen and Start/End TimeStamps
 
-=== Audit process and Benchmark Logging
+The MLPerf Power workflow uses exactly the same LoadGen as used for performance runs.
+LoadGen logs the system timestamp at the start and at the end of a performance run.
+The workflow then uses these timestamps to evaluate the power consumption of the run.
 
-As part of the submissions and logging, all the logs generated by the
-MLPerf Power SW infrastructure need to be submitted. These include the
-power meter ranging logs and the power measurement logs that are
-generated during the performance runs.
+=== Power and Performance Measurement and Reporting
 
-=== Replicability is mandatory
+Power and performance measurements should be from the same run for a
+given benchmark and scenario. The MLPerf Power workflow takes care of this by
+default. This must not be changed. Example: It is not permitted to run
+the same benchmark and scenario 3 times and report the highest
+performance and the lowest power consumption among the 3 runs.
 
-Results that cannot be replicated are not valid results.
+=== Realistic Power Management
 
-=== Power and Performance Reporting and Measurement
+The goal of the testing is to mimic real-world usage scenarios as much
+as possible and enable showing the benefits of realistic power
+management. Therefore, we require that:
 
-Power and performance measurements should be from the same run for a
-given benchmark and scenario. The current script takes care of this by
-default and it cannot and should not be changed. Example: We cannot run
-the same benchmark and scenario 3 times and report the highest
-performance and lowest power among the 3 runs.
+* Any power management system be qualified for use appropriate for the submission type (e.g., a generally available system must use software/firmware qualified for general availability and shipping with the platform).
+* No benchmark- or benchmarking-specific hacks.
+* Any changes in power management behavior must not have manual intervention or have awareness of the benchmark.
 
-== Power Analyzer Support
 
-For version 1.0 and version 1.1 , we will only support Yokogawa power analyzers (aka meters).
+== What needs to be submitted?
 
-== Power Analyzer settings
+=== Platform Description
 
-The power analyzer settings will not be set manually, but through the
-software that is part of the MLPerf Power measurement infrastructure.
+A valid submission must have all the mandatory fields of the SUT description
```

Comment:
@psyhtest  -  We want to state that all the mandatory fields of the SUT description tied to Power.  For Power submission we will have additional components that will be desired to be mandatory over and above the Perf Only submission

Reply:
Agree. We probably should define these components in these rules as well. Perhaps at a next iteration?